types/pixi/doorControl.d.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * An icon representing a Door Control
+ */
+declare class DoorControl extends PIXI.Container {
+  constructor(wall: Wall);
+
+  /**
```

Comment:
@am-a @ghost91- @BoltsJ I think we need to have this discussion sooner or later.  This is currently missing the separator line comment that the foundry source has between almost all methods.

I am seeing three variants here:
- copy everything 1:1 from the original source
- omit the separator lines and copy only the section headers
- omit all of them

I am fine with any of those, but it **has** to be consistent. I don't want to have one merge request not having them and another does. Whatever we decide, we should probably also formalize in some document in the project root.

Reply:
I don't really have a preference. Personally, as a user of the types, I think there is not much value in having any of these separators / headings in the type definitions. However, it might make it easier to keep them in sync with foundry.js. I'm fine with all of the options. But I agree that it should be consistent.

Reply:
After some discussion we all agreed that we leave out the non-doc comments. That would mean you can also remove the heading comments.